### ISSUE TITLE
evaluate fine-tuned model

### ISSUE THREAD

**Issue by bwanglzu** [LAION-AI/CLIP_benchmark#13]
hi guys, great work!

I'm currently tying to use varies CLIP to evaluate how finetuning would improve the search results on varies IR metrics.

i would like to build something on top of the zero-shot retrieval results here :) while i realized you're using varies datasets to evaluate the zero-shot CLIP. My question is are you using the whole dataset for evaluation? if , let's say i want to fine-tune CLIP on flowers dataset, i have to split training/evaluation/test set

**Comment by bwanglzu** [comment]
ignore, i see the `split` option when build the dataset.

**Comment by mehdidc** [comment]
Hi, indeed, there is a `split` option and we use the `test` split by default. Note that for some Torchvision based datasets `split` is ignored (e.g., eurosat and caltech101 ) as it is not supported so the full dataset is used, there it is better to use tfds/ or VTAB/ datasets (e.g., `vtab/eurosat` and `vtab/caltech101` as dataset names), where splits are in general available (e.g. for VTAB, you can look at split names here https://github.com/google-research/task_adaptation/tree/master/task_adaptation/data).

**Comment by mehdidc** [comment]
Yes of course, please do!

**Comment by bwanglzu** [comment]
hi @mehdidc these are my initial results, still working on clean up the codebase :)

![image](https://user-images.githubusercontent.com/9794489/194298506-1658c0bf-d09c-4fbf-beac-d30cc324dc62.png)

i'm a bit confused by flickr8k (why improve so much) and coco (why only little improvement on image recall), still need some investigation.
